week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,37 +1,17 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
```

Comment:
The first two are looking great here, the test case is failing for the third - can you take a look :)